packages/thyra/adapters/tpetra/src/Thyra_TpetraThyraWrappers_def.hpp

```diff
@@ -152,7 +167,9 @@ Thyra::createMultiVector(
     getOrCreateTpetraVectorSpace(rangeSpace, tpetraMultiVector_in->getMap()),
     getOrCreateLocallyReplicatedTpetraVectorSpace<Scalar, LocalOrdinal, GlobalOrdinal, Node>(
       domainSpace, tpetraMultiVector_in->getMap()->getComm(),
+#ifdef TPETRA_ENABLE_DEPRECATED_CODE
       tpetraMultiVector_in->getMap()->getNode(),
```

Comment:
I don't think you need to call the deprecated overload any more, even if deprecated code is enabled.  Ditto for other instances below.

Reply:
Calling the deprecated overload is the only way to test that the overload is correct (or at least compiles).  We learned this the hard way with the xpetra deprecations.